js/ui/form/ui.form.layout_manager.js

```diff
@@ -542,6 +542,31 @@ var LayoutManager = Widget.inherit({
             .html("&nbsp;");
     },
 
+    _getButtonHorizontalAlignment: function(item) {
+        if(typeUtils.isDefined(item.horizontalAlignment)) {
+            return item.horizontalAlignment;
+        }
+
+        if(typeUtils.isDefined(item.alignment)) {
+            var formInstance = this.option("form");
+            errors.log("W0001", formInstance && formInstance.NAME, "alignment", "18.1", "Use the 'horizontalAlignment' option in button items instead.");
```

Comment:
If LayoutManager can operate with button items independently, you shouldn't use the Form-specific error message. Can we make the message more neutral?

Reply:
This message for DateBox - 
W0001 - dxDateBox - 'maxZoomLevel' option is deprecated in 18.1. Use the 'calendarOptions.maxZoomLevel' option instead. See:
http://js.devexpress.com/error/18_1/W0001
This our template for deprecated options.

Reply:
I suggest to use a plain text instead of using `formInstance` (that can be undefined).
Like [there](https://github.com/DevExpress/DevExtreme/blob/18_1/js/ui/toolbar/ui.toolbar.strategy.js#L75)